### DISCUSSION TITLE
Netxtcloud Office language

### DISCUSSION THREAD

**Discussion by chamabreu** [nextcloud/all-in-one#1371] [Questions] [answered]
Hi,

I like to use Nextcloud in English, but I want Nextcloud Office to be in German.
Thats because I need German formatting for Dates.

If i open a Document from a Nextcloud   Account which is set to Geman, the Office Application is also in german, so are the dates formatted.
If i open the same Document then on a Nextcloud Account which is set to English, the Office Application is in English and so are the Dates.

Is there a posibility to change that?
In Nextcloud Office there is under "Tools" a menu with language, but this has no effect.

**Comment by szaimen** [comment, marked as answer]
This is currently not possible, afaik. Feel free to open a feature request here if it should not exists yet: https://github.com/nextcloud/richdocuments/issues
